docs/source/traitsui_user_manual/testing/substitutions.rst

```diff
@@ -0,0 +1,38 @@
+
+..
+   # substitutions
+
+.. |testing.api| replace:: :mod:`traitsui.testing.api`
+
+.. |Group| replace:: :class:`~traitsui.group.Group`
+.. |Item| replace:: :class:`~traitsui.item.Item`
+.. |UI| replace:: :class:`~traitsui.ui.UI`
+
+.. |MouseClick| replace:: :class:`~traitsui.testing.tester.command.MouseClick`
+.. |KeySequence| replace:: :class:`~traitsui.testing.tester.command.KeySequence`
+.. |DisplayedText| replace:: :class:`~traitsui.testing.tester.query.DisplayedText`
+.. |Index| replace:: :class:`~traitsui.testing.tester.locator.Index`
+.. |TargetById| replace:: :class:`~traitsui.testing.tester.locator.TargetById`
+.. |TargetByName| replace:: :class:`~traitsui.testing.tester.locator.TargetByName`
+
+.. |TargetRegistry| replace:: :class:`~traitsui.testing.tester.target_registry.TargetRegistry`
+.. |TargetRegistry.register_interaction| replace:: :func:`~traitsui.testing.tester.target_registry.TargetRegistry.register_interaction`
+.. |TargetRegistry.register_location| replace:: :class:`~traitsui.testing.tester.target_registry.TargetRegistry.register_location`
+
+.. |UITester| replace:: :class:`~traitsui.testing.tester.ui_tester.UITester`
+.. |UITester.auto_process_events| replace:: :attr:`~traitsui.testing.tester.ui_tester.UITester.auto_process_events`
+.. |UITester.create_ui| replace:: :func:`~traitsui.testing.tester.ui_tester.UITester.create_ui`
+.. |UITester.delay| replace:: :attr:`~traitsui.testing.tester.ui_tester.UITester.delay`
+.. |UITester.find_by_id| replace:: :func:`~traitsui.testing.tester.ui_tester.UITester.find_by_id`
+.. |UITester.find_by_name| replace:: :func:`~traitsui.testing.tester.ui_tester.UITester.find_by_name`
+.. |UIWrapper| replace:: :class:`~traitsui.testing.tester.ui_wrapper.UIWrapper`
+.. |UIWrapper.find_by_id| replace:: :func:`~traitsui.testing.tester.ui_wrapper.UIWrapper.find_by_id`
+.. |UIWrapper.find_by_name| replace:: :func:`~traitsui.testing.tester.ui_wrapper.UIWrapper.find_by_name`
+.. |UIWrapper.help| replace:: :func:`~traitsui.testing.tester.ui_wrapper.UIWrapper.help`
+.. |UIWrapper.inspect| replace:: :func:`~traitsui.testing.tester.ui_wrapper.UIWrapper.inspect`
+.. |UIWrapper.locate| replace:: :func:`~traitsui.testing.tester.ui_wrapper.UIWrapper.locate`
+.. |UIWrapper.perform| replace:: :func:`~traitsui.testing.tester.ui_wrapper.UIWrapper.perform`
+.. |UIWrapper._target| replace:: :attr:`~traitsui.testing.tester.ui_wrapper.UIWrapper._target`
+
+.. |ModalDialogTester| replace:: :class:`~pyface.ui.qt4.util.modal_dialog_tester.ModalDialogTester`
```

Comment:
Aha, I am guessing this did not link to docs because it is a toolkit specific class

Reply:
Yep.